functions/Get-SqlIndexes.ps1

```diff
@@ -0,0 +1,107 @@
+﻿Function Get-SqlIndexes
```

Comment:
We have now moved to the -Dba prefix and PowerShell likes its singular function and param names, so this would have been Get-DbaIndexInfo (The info is added because it's not an Index SMO object, but rather, a query.